src/utilities/formatters.js

```diff
@@ -49,3 +49,17 @@ export const formatTemperatureExposure = ({ minimumTemperature, maximumTemperatu
 
   return `${formatTemperature(minimumTemperature)} - ${formatTemperature(maximumTemperature)}`;
 };
+
+export const formatTimeDifference = duration => {
+  const suffixes = [generalStrings.days, generalStrings.hours, generalStrings.minutes];
+  const durations = [duration.days(), duration.hours(), duration.minutes()];
+
+  // merges the above arrays to give a string for example: 3 days, 4 hours, 3 minutes
+  return durations.reduce(
+    (acc, value, index) =>
+      value
+        ? `${acc} ${value} ${suffixes[index]}${durations.length - 1 === index ? '' : ','}`
+        : acc,
+    ''
+  );
+};
```

Comment:
is this confusing?

Takes a duration moment object and formats it so its like "3 days, 4 minutes" or "4 hours, 6 minutes" or "2 days, 7 hours" etc etc

Reply:
> confusing?

No - I thought that it was clear enough. Wondered if `moment` would take care of it - that would be tricky to handle the `0` though, so stick with what you have!